docs/markdown/pipelines/exomeseq.md

```diff
@@ -42,106 +58,127 @@ Running WGS analysis with Sarek does not require any additional files.
+  ```
 
-  - `hg19` & `Twist capture kit`
+- `hg19` & `Twist capture kit`
 
-    ```txt
-    /nfsmounts/igenomes/Homo_sapiens/UCSC/hg19/Exome_BED/Twist_Exome_Target_hg19_noUn_gl000228_2020_05_05.bed
-    ```
+  ```txt
+  /nfsmounts/igenomes/Homo_sapiens/UCSC/hg19/Exome_BED/Twist_Exome_Target_hg19_noUn_gl000228_2020_05_05.bed
+  ```
 
-  - Mouse `mm10` & `Agilent SureSelect All Exon`
+- Mouse `mm10` & `Agilent SureSelect All Exon`
 
-    ```txt
-    /nfsmounts/igenomes/Mus_musculus/UCSC/mm10/Exome_BED/mm10_Agilent_SureSelect_Mouse_All_Exon_Kit_2016_01_21_sorted.bed
-    ```
+  ```txt
+  /nfsmounts/igenomes/Mus_musculus/UCSC/mm10/Exome_BED/mm10_Agilent_SureSelect_Mouse_All_Exon_Kit_2016_01_21_sorted.bed
+  ```
 
-  - Mouse `GRCm38` & `Agilent SureSelect Mouse`
+- Mouse `GRCm38` & `Agilent SureSelect Mouse`
 
-    ```txt
-    /nfsmounts/igenomes/Mus_musculus/Ensembl/GRCm38/Exome_BED/GRCm38_Agilent_SureSelect_Mouse_All_Exon_Kit_2016_01_21_sorted.bed
-    ```
+  ```txt
+  /nfsmounts/igenomes/Mus_musculus/Ensembl/GRCm38/Exome_BED/GRCm38_Agilent_SureSelect_Mouse_All_Exon_Kit_2016_01_21_sorted.bed
+  ```
 
-- Notes on Target BEDs: In general for WES data analysis people advise to use target regions that are padded (e.g. +- 150 bp), also the Sarek docs said that. But to be precise there are multiple aspects:
-  - The mapping should be done on the whole reference to get information about multi reads mapping also outside of the exome (to discard)
-  - Then reads mapping within the target exome regions should be kept, for this a padding should be added to prevent loosing reads at the boarders
-  - for QC to get an unbiased view about the overall quality of the experiment its best to use the exact target regions without padding
-  - and for the final variants it depends, if one want to be sure not to increase the number of FPs it would also make sense to use unpadded target regions. If the goal would be to get as many variants as possible, then of course one could use padded regions (which made quite a difference in the final numbers in my case)
-  - Sarek actually maps reads against the whole reference, and then uses the targetBED file for the QC and to filter variants. So _simply hand over the unpadded target region_. This is usually provided by the Exon capture kit company.
+In general for WES data analysis people advise to use target regions that are padded (e.g. +- 150 bp). But to be precise there are multiple aspects:
+
+- The mapping should be done on the whole reference to get information about multi reads mapping also outside of the exome (to discard). This is done by default by the pipeline, even when the target bed file is provided.
+- Then reads mapping within the target exome regions should be kept, for this a padding should be added to prevent loosing reads at the boarders. This is done by default by the pipeline for base quality score recalibration and variant calling.
```

Comment:
```suggestion
- Reads mapping within the target exome regions should be kept. For this a padding should be added to prevent loosing reads at the borders. This is done by default by the pipeline for base quality score recalibration and variant calling.
```